js/languages/en-US.json

```diff
@@ -551,7 +553,7 @@
     "pageUnavailable": "This page is currently unavailable.",
     "badHandle": "The handle you entered does not have a valid OpenBazaar ID.",
     "serverDown": "The server has been shut down",
-    "unableToLoadImages": "We were unable to load your image(s).",
+    "unableToLoadImages": "We were unable to load your image.  |||| We were unable to load your images.",
```

Comment:
This doesn't look right..? You're calling it and passing in a `smart-count`, which doesn't appear to be used. Also, should those four vertical lines really be there (is there some weird polyglot plural/singular syntax I'm blissfully oblivious to?)